src/SignalR/clients/ts/signalr/src/HubConnectionBuilder.ts

```diff
@@ -13,6 +13,32 @@ import { JsonHubProtocol } from "./JsonHubProtocol";
 import { NullLogger } from "./Loggers";
 import { Arg, ConsoleLogger } from "./Utils";
 
+// This is exported from this module but NOT from index, so it's "internal" for testing purposes
+export const LogLevelNameMapping = {
+    // TS Lint prefers alphabetical sorting...
```

Comment:
Put `/* tslint:disable:object-literal-sort-keys */` above the declaration and sort based log severity.